q.sql

```diff
@@ -0,0 +1,160 @@
+--Create a table students and insert names in malayalam
+CREATE TABLE STUDENTS(
+STUD_ID INT PRIMARY KEY,
+NAME NVARCHAR(30) NOT NULL
+);
+
+INSERT INTO STUDENTS VALUES(02,N'റാം');
```

Comment:
What will be the STUD_ID value of employee "റാം"

Reply:
![image](https://github.com/user-attachments/assets/728e995a-7ae8-47f8-9a2a-458f9b2c9cf7)

STUD_ID = 2